src/node_sockaddr.h

```diff
@@ -116,6 +117,40 @@ class SocketAddress : public MemoryRetainer {
   sockaddr_storage address_;
 };
 
+template <typename T>
+class SocketAddressLRU : public MemoryRetainer {
+ public:
+  using Type = typename T::Type;
+  using Pair = std::pair<SocketAddress, Type>;
+  using Iterator = typename std::list<Pair>::iterator;
```

Comment:
These should go into the `private:` section as implementation details, imo?

Reply:
Yeah, they definitely can. Will move them